src/main/resources/application.properties

```diff
@@ -2,13 +2,13 @@ spring.application.name=KafkaStudy
 kafka.regular.topic=my-topic
 regular.kafka.autostart=false
 reactive.kafka.autostart=true
-spring.kafka.consumer.bootstrap-servers=127.0.0.1:9092
+spring.kafka.consumer.bootstrap-servers=kafka:9092
 spring.kafka.consumer.group-id=demo-3
 spring.kafka.consumer.auto-offset-reset=earliest
 spring.kafka.consumer.key-deserializer=org.apache.kafka.common.serialization.StringDeserializer
 spring.kafka.consumer.value-deserializer=org.apache.kafka.common.serialization.StringDeserializer
 spring.kafka.consumer.properties.session.timeout.ms=30000
 spring.kafka.consumer.properties.heartbeat.interval.ms=10000
-spring.data.mongodb.uri=mongodb://root:example@localhost:27017/testMongoDb?authSource=admin
+spring.data.mongodb.uri=mongodb://root:example@mongodb:27017/testMongoDb?authSource=admin
```

Comment:
Data successfully saved to MongoDB.
![image](https://github.com/user-attachments/assets/da53b07b-64a8-4ae7-ae01-2471965ed902)